src/analytics.ts

```diff
@@ -62,8 +75,12 @@ const withAnalytics = ({
     ) {
         // Call iOS interface
         return onIos(window.webkit.messageHandlers.firebase);
+    } else if (
+        window.ga &&
+        // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
```

Comment:
can't we just remove this lint rule?

Reply:
Yes, we can. It's like a linting to remind to don't avoid linting rules xD
I'll override it